### ISSUE TITLE
Rethink stopps on it's own

### ISSUE THREAD

**Issue by muellerto** [celzero/rethink-app#937]
Again and again I find that Rethink is stopped and I swear I didn't do that. So how come?

In the most cases I detect that an app which should be blocked from network access is suddenly able to do that access. When I then look for the app's settings in Rethink I see that "I'm not protected" at all. And when that happens I never know if this came just three minutes before or if it already was the case for a week.

I use v054c from F-Droid on /e/ 1.11 (Android 12). I would say earlier versions of Rethink didn't do that.

**Comment by ignoramous** [comment]
Hi:

The app is crashing on your Android. This can happen due to a number of reasons:
- Insufficient RAM / RAM pressure on your device (Games may end up killing Rethink and other apps).
   - Can't do much other than buy an Android device with a bigger RAM or not use RAM hogging apps.
- Battery optimizer killing long running processes such as Rethink
   - Exempt Rethink from [Battery Optimizations](https://dontkillmyapp.com/).
- Crashes
   - Email (`mz` at `celzero` dot `com`) the output from [`adb bugreport`](https://developer.android.com/studio/debug/bug-report), if you're comfortable doing so.

In the meanwhile, if you want to, then the previous version of the app, if those were stable enough for you, are available for install from both [F-Droid](https://f-droid.org/en/packages/com.celzero.bravedns/) and [our Website](https://rethinkdns.com/download).

**Comment by muellerto** [comment]
Can't imagine a RAM exhaustion. I never run games. Most used apps are Signal, FairEmail, Music player, Camera and Browser (Mull). And all these apps I had for two years.

I set the app's accu usage now on "not optimized". Let's see if this has an effect.

Because of a problem after the change from 053 to 054 (054 didn't come up anymore and clearing the cache didn't help, the same happened on another device) I removed and reinstalled it completely. Sure I didn't change the accu optimization after that.
